README.md

```diff
@@ -41,15 +41,35 @@ import {
   closeRedis
 } from "@myunisoft/redis";
 
-const redis = await initRedis();
+const publisher = await initRedis();
+const subscriber = await initRedis({}, true);
 
-assert.strictEqual(redis, getRedis());
+assert.strictEqual(publisher, getPublisher());
+assert.strictEqual(subscriber, getSubscriber());
 
-await closeRedis();
+await closeAllRedis();
 ```
 
 ## 📜 API
 
+### getPublisher(): Redis;
+
+> This function return the Redis publisher instance.
+
+---
+
+### getSubscriber(): Redis;
+
+> This function return the Redis subscriber instance.
+
+---
+
+### initRedis(redisOptions: Partial<RedisOptions> & { port?: number; host?: string; } = {}, initSubscriber?: boolean): Promise<Redis>
+
+> This function is used to init redis connections, passing extInstance at true, init the subscriber local instance, otherwise, it init the publisher local instance.
```

Comment:
```suggestion
This function is used to initialize Redis connections. If the extInstance parameter is set to true, it initializes the local instance as a subscriber. Otherwise, it initializes the local instance as a publisher.
```
I think we can improve the ponctuation there, wdyt ?